lib/fetching/bots/pull-bot.js

```diff
@@ -8,7 +8,7 @@ var logger = require('../../logger');
 // options.source - The source to pull from.
 // options.contentService - The contentService to control.
 var PullBot = function(options) {
-  this.interval = options.interval || 30000;
+  this.interval = options.contentService.interval || 120000;
```

Comment:
I might be missing the context but what was the reason for this change?

Reply:
The original interval was 120000, so I returned it to the default and set up a way to configure the interval via a content provider so that CrowdTangle can set the interval to 30000.